src/components/sectionList/SectionListWrapper.tsx

```diff
@@ -97,9 +131,10 @@ export const SectionListWrapper = ({
                 model={model}
                 onShowDetailsClick={handleDetailsClick}
                 onOpenSharingClick={setSharingDialogId}
+                onDeleteClick={() => deleteModel(model.id)}
```

Comment:
I think we might need a confirm deletion modal? Not sure

Reply:
CC @cooper-joe

Reply:
Yes, I think so (unless we can have undo ;) )

Reply:
I've added the confirmation modal:


https://github.com/dhis2/maintenance-app-beta/assets/10101831/21320b64-f848-47a1-b1dd-3acbc8a2cf9a

I think we should also display that something is being done in the background (otherwise you might be asking yourself if the click on "Yes" actually did anything unitl you get the alert at the bottom of the screen)